docs/templates/sec10k_child.rst.jinja

```diff
@@ -0,0 +1,431 @@
+captured from the Q1 filings for 2018-2022. Approximately 80% of companies make their
+annual SEC 10-K filings in Q1, so this results in a substantial data gap in those years
+which may also affect the quality of the record linkage. See issue :issue:`4165` and PR
+:pr:`4134`. We should be able to fill in these gaps by re-running the extraction
+pipeline.
+
+We don't attempt to capture nested subsidiary relationships
+-----------------------------------------------------------
+
+Exhibit 21 attachments often present subsidiary relationships in a nested format,
+using indentation to indicate ownership hierarchies. Our dataset does not attempt to
+capture this nested structure directly. Instead, if an Exhibit 21 lists multiple
+layers of subsidiaries, we extract all subsidiaries as direct children of the parent
+company that filed the Exhibit 21. In practice, many of the intermediate subsidiaries
+also file their own 10-K reports, where they list their own subsidiaries, and which
+can be used to infer the full ownership hierarchy.
+
+Matches between EIA utility ids and SEC 10-K energy companies are sparse
+------------------------------------------------------------------------
+
+.. figures in this section derived in notebooks/work-in-progress/sec10k_irregularities_figures.ipynb
+
+Only 14% of EIA utilities are linked to an SEC filer or subsidiary
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ideally, we would be able to link most companies with EIA ids to either a parent or a
+subsidiary company listed in the SEC 10-K.
+
+As of PUDL release v2025.9.1, only 2,308 out of 16,636 utilities with EIA ids have been
+identified in the SEC data; just under 14%. While these are all high-confidence matches,
+the link rate may not be enough for many desired use cases.
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - EIA utility IDs
+     - Count
+   * - All
+     - 16,636
+   * - Mapped to an SEC 10-K filer central index key
+     - 529
+   * - Mapped to an SEC 10-K subsidiary
+     - 1,940
+
+Maps to an SEC 10-K filer central index key are superior, since the data we have about filers
+is significantly richer than the data we have about subsidiaries.
+
+
+Only half of SEC filers providing electric services are linked to EIA utilities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ideally, we would be able to link most SEC companies that are clearly utilities to their
+associated EIA utility id.
+
+This is a little trickier to evaluate. SEC filers use a four-digit Standard Industrial
+Classification (SIC) code to identify the company's primary industry, but not all utilities
+file under the SICs for electric services (4911 and 4931), and not all companies that
+file as electric services are utilities. We see SICs as unexpected as computer storage
+devices, nursing facilities, and real estate among our -- high-confidence -- matches to EIA
+utilities.
```

Comment:
An intuitive expectation that I have based on what I know about the SEC 10-K filers and browsing through the companies listed under SICs 4911 and 4931 is that conceptually, a very high proportion of the filers in those industries should really show up in the set of EIA utilities.  I think to a much higher degree than I would expect that the EIA utilities would necessarily show up as SEC 10-K filers, since there are so many smaller special-purpose companies that exist only to own a single generator or slices of some plant, or to act as an operator without owning the underlying asset that they're operating.